docs/index.html

```diff
@@ -10,7 +10,10 @@
     <!-- Main jumbotron for a primary marketing message or call to action -->
     <div class="jumbotron">
       <div class="container">
-        <img src="{{ site.baseurl }}/guides/ForemastHighLevel.svg" alt="Foremast High Level Overview">
+        <!-- <img src="{{ site.baseurl }}/guides/ForemastHighLevel.svg" class="img-fluid" alt="Foremast High Level Overview"> -->
```

Comment:
@srivathsanvc: I commented out this image simply to highlight the capability of adding a responsive video container

though, I also added a new class: `img-fluid` which comes free in bootstrap and makes a non-responsive image responsive

you'll probably want to comment out the video until we've gone one of our own to use prior to re-deployment